### PR TITLE
Lint Markdown files

### PR DESCRIPTION
**Pull request by clementbiron**
Fixes https://github.com/OpenTermsArchive/docs/issues/5

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Node
 node_modules
 npm-debug.log
-package-lock.json
```
Comment (MattiSG) [inline .gitignore:4]: 🥷

This is a “ninja commit” should be done in a different PR 🙂

File: package.json [hunk 2]
```diff
@@ -13,6 +13,10 @@
   "license": "CC BY-SA 4.0",
   "author": "Open Terms Archive",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (MattiSG) [inline package.json:16]: Replace `test` by `lint:markdown`, or at least run `lint:markdown` through `test` 🙂

File: package.json [hunk 3]
```diff
@@ -13,6 +13,10 @@
...
+    "lint:markdown": "markdownlint *.md content/*.md content/**/*.md content/**/**/*.md "
```
Comment (MattiSG) [inline package.json:17]: This is redundant. The glob syntax should be enough for traversal:

```suggestion
    "lint:markdown": "markdownlint **/*.md"
```

**Review by MattiSG — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (clementbiron) [reply package.json]: Ok I replaced `test` by `lint:markdown`.

File: package.json (hunk 3 above)
Comment (clementbiron) [reply package.json]: `markdownlint **/*.md` results in linting all markdown files from the repository which is long and unnecessary (the `node_modules` folder for example!) and does not lint the files in the root path (like the `README.md`).

So I changed by `markdownlint *.md content/**/*.md` 👍

File: .gitignore (hunk 1 above)
Comment (clementbiron) [reply .gitignore]: Ok I made a fixup here and that will be bone by merging https://github.com/OpenTermsArchive/docs/pull/13 before.

File: package.json [hunk 4]
```diff
@@ -13,6 +13,9 @@
   "license": "CC BY-SA 4.0",
   "author": "Open Terms Archive",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint:markdown": "markdownlint *.md content/**/*.md"
```
Comment (MattiSG) [inline package.json:16]: Sorry, I was not clear. It is very useful to have the `test` script, as it is one of the “lifecycle” scripts of NPM, which means it has special meaning (for example, if you write `npm test` it would work, you don't need to write `npm run test`). By “replacing”, I meant replacing the contents of the script 🙂 

```suggestion
    "test": "markdownlint *.md content/**/*.md"
```

You could also do the following if you want to keep an explicit name for subtasks:

```suggestion
    "lint:markdown": "markdownlint *.md content/**/*.md",
    "test": "npm run lint:markdown"
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
This will need a rebase on top of #13 to pass the build.

File: package.json (hunk 4 above)
Comment (clementbiron) [reply package.json:17]: Thanks for the clarification, I did a fixup.

**Review by MattiSG — APPROVED** [review]
